app/models/user/User.scala

```diff
@@ -182,13 +181,10 @@ object UserDAO extends SecuredBaseDAO[User] {
 
   def findByIdQ(id: BSONObjectID) = Json.obj("_id" -> id)
 
+  /*
   def authRemote(email: String, loginType: String)(implicit ctx: DBAccessContext) =
```

Comment:
what is authRemote and why is it commented out? should we delete it?

Reply:
This is never used (even before silhouette). I removed it.